tgui/packages/tgui/interfaces/IssueReporter.js

```diff
@@ -0,0 +1,226 @@
+import { Window } from '../layouts';
+import { useBackend, useLocalState } from '../backend';
+import { ButtonCheckbox } from 'tgui/components/Button';
+import { Box, Section, Button, Flex, TextArea, Input } from '../components';
+
+export const IssueReporter = (props, context) => {
+
+  const {
+    act,
+  } = useBackend(context);
+
+  const [
+    isRegression,
+    setIsRegression,
+  ] = useLocalState(context, 'isRegression', false);
+  const [
+    issueName,
+    setIssueName,
+  ] = useLocalState(context, 'issueName', '');
+  const [
+    replicationSteps,
+    setReplicationSteps,
+  ] = useLocalState(context, 'replicationSteps', '');
+  const [
+    expected,
+    setExpected,
+  ] = useLocalState(context, 'expected', '');
+  const [
+    actual,
+    setActual,
+  ] = useLocalState(context, 'actual', '');
+
+  const [
+    stage,
+    setStage,
+  ] = useLocalState(context, 'stage', 1);
+
+  let content = "The issue reporter is currently facing issues.";
+  let windowHeight = 480;
+
+  switch (stage)
+  {
+    case 1:
+      content = (<Regression
```

Comment:
don't do switches with renderable elements in them like this. it breaks the fuck out of TGUI and makes weird errors.
You'll have to do something like this https://github.com/BeeStation/BeeStation-Hornet/blob/30e8bf74ded22ebcf88e77464bd6d228feebaeec/tgui/packages/tgui/interfaces/TraitorObjectivesMenu.js#L20-L43